Utilities/Log.cpp

```diff
@@ -1,4 +1,4 @@
-﻿#include "Log.h"
+#include "Log.h"
```

Comment:
How you managed to damage unicode mark?

Reply:
Couldn't tell you , I'm just using Visual Studio community 2017 on Windows. Would you like me to do another commit to try and remove these kind of commits (Removed X and added X) and remove some unintended changes like newlines being removed?

Reply:
No, I pushed fixes in my PR